Shared/dugga.js

```diff
@@ -565,8 +557,8 @@ function randomPassword()
 //----------------------------------------------------------------------------------
 
 function createUrl(hash) {
-	var realUrl = window.location + "?a=" + hash;
-	var localhostUrl="http://localhost/LenaSYS/sh/?a=" + hash;
+	var realUrl = window.location + "&hash=" + hash;
+	var localhostUrl="http://localhost/LenaSYS/DuggaSys/sh/?hash=" + hash;
```

Comment:
Possible that these changes are wrong since sh/index.php does not currently get the value trough the parameter hash but rather the parameter a. So if this change is introduced then it's possible that index.php would work correctly in it's current state when redirected to by dugga.js.

Reply:
Anything related to hashes in in this pull request have been added by mistake somehow, they need to be removed/discarded before a merge is done.